config/gatekeeper.yml

```diff
@@ -10,6 +10,8 @@ localhost:
     - 1K022G # portal dev/stg test user
     - 2A555X # portal dev/stg test user
     - 3B345C # portal dev/stg test user which is inactive
+    - 64dc8505-5fd6-493b-bacb-5d3020364001
```

Comment:
This is temporarily required whilst we wait for LASSIE to include the office code rather than the office id.